src/main/java/com/ericsson/ei/queryservice/ProcessQueryParams.java

```diff
@@ -37,7 +36,7 @@
 @Component
 public class ProcessQueryParams {
```

Comment:
This class should be renamed. Please discuss within the team.

Reply:
Could that be done during a joint refactoring session though?

Reply:
Then we could also discuss the naming of this package. Right now the package is called "queryservice" (which I always confuse with "erqueryservice"). But other services are located in the "services" package ... 🤔  So why do we name this package queryservice? Maybe it should called something else instead.